js/main.ts

```diff
@@ -0,0 +1,108 @@
+const enum Default {
+	ARRAY_LENGTH = 25,
+	LENGTH_MIN = 0,
+	LENGTH_MAX = 30
+}
+
+const enum Img {
+	IMG_URL_MIN = 1,
+	IMG_URL_MAX = 25,
+	IMG_ID_MIN = 1,
+	IMG_ID_MAX = 25,
+}
```

Comment:
Мне кажется ты переусложняешь
```suggestion
const enum Img {
	ID_MIN = 1,
	ID_MAX = 25,
}
```